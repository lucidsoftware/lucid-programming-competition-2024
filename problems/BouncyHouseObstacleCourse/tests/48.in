3
A
B
C
3
A B 1
B C 1
A C 2
