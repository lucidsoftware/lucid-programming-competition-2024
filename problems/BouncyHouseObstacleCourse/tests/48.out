4971232
0
40
116
192
193
241
308
322
409
491
582
630
702
747
810
878
955
1029
1128
1225
1302
1401
1485
1577
1660
1739
1829
1924
1992
2082
2169
2248
2346
2431
2518
2606
2695
2787
2883
2980
3080
3166
3255
3344
3409
3506
3603
3701
3794
3894
3991
4085
4181
4268
4350
4440
4538
4623
4721
4815
4915
5009
5108
5208
5305
5403
5502
5593
5679
5779
5878
5972
6048
6138
6214
6309
6394
6456
6531
6627
6705
6798
6874
6972
7065
7164
7263
7357
7455
7538
7623
7723
7805
7898
7988
8076
8145
8245
8345
8412
8510
8607
8679
8767
8863
8949
9042
9141
9236
9326
9425
9518
9613
9707
9786
9886
9967